dev/com.ibm.websphere.appserver.features/visibility/public/mail-2.0/io.openliberty.mail-2.0.feature

```diff
@@ -4,8 +4,9 @@ visibility=public
 singleton=true
 IBM-ShortName: mail-2.0
 Subsystem-Version: 2.0
-Subsystem-Name: mail 2.0
+Subsystem-Name: Jakarta mail 2.0
```

Comment:
This should conflict with my PR that I merged because It is `Jakarta Mail 2.0` instead of Jakarta mail 2.0`.

Reply:
It just may be easier to undo the subsystem name and rebase on integration